### PR TITLE
Create initial android project from wizard

### PR DESCRIPTION
**Pull request by arist0v**
Initial empty compose project

### CODE REVIEW

**Review by benfrancis — COMMENTED** [review]
It's a few years since I created an empty Android project so I'm not really sure what most of this boilerplate configuration is for, so the only helpful feedback I have is on naming 😆

Please note that the name of the project should be capitalised as "WebThings", not "Webthings".

Regarding the java namespace, `io.webthings.webthings` is a bit confusing because of the repetition and the potential for confusion with the `io.webthings.webthing` namespace used by the [webthing-java](https://github.com/WebThingsIO/webthing-java) library. Would `io.webthings.app` make more sense? @tim-hellhake what do you think?

**Comment by arist0v** [comment]
i agree to move it to app instead of second webthings, for now the major part of those file isn't OK since it was the default android compose project settings, i will adapt the file in the next PR i will do

**Review by tim-hellhake — CHANGES_REQUESTED** [review]
The `.idea` folder shouldn't be committed to the repo.
Please add it to `.gitignore` and remove the it.

**Comment by tim-hellhake** [comment]
> Would `io.webthings.app` make more sense? @tim-hellhake what do you think?

I agree.

**Comment by arist0v** [comment]
> > Would `io.webthings.app` make more sense? @tim-hellhake what do you think?
> 
> I agree.

This will be changed in #49

**Comment by tim-hellhake** [comment]
I would prefer to change it here.
Otherwise, it will add a lot of noise to other PRs.

File: .gitignore [hunk 1]
```diff
@@ -38,6 +38,7 @@ captures/
 
 # IntelliJ
 *.iml
+.idea/
 .idea/workspace.xml
 .idea/tasks.xml
 .idea/gradle.xml
```
Comment (tim-hellhake) [inline .gitignore:44]: Those are no longer needed.
```suggestion
```

**Review by tim-hellhake — CHANGES_REQUESTED** [review]

**Review by tim-hellhake — CHANGES_REQUESTED** [review]
Please change the namespace to `io.webthings.app` then we can merge it.

**Comment by arist0v** [comment]
done

**Comment by tim-hellhake** [comment]
@arist0v You don't need to replicate the changes in the other PRs.
As soon this is merged you can rebase the other branches.

File: app/src/androidTest/java/io/webthings/app/ExampleInstrumentedTest.kt [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+ *
+ * See [testing documentation](http://d.android.com/tools/testing).
+ */
+@RunWith(AndroidJUnit4::class)
+class ExampleInstrumentedTest {
+    @Test
+    fun useAppContext() {
+        // Context of the app under test.
+        val appContext = InstrumentationRegistry.getInstrumentation().targetContext
+        assertEquals("io.webthings.webthings", appContext.packageName)
```
Comment (tim-hellhake) [inline app/src/androidTest/java/io/webthings/app/ExampleInstrumentedTest.kt:22]:
```suggestion
        assertEquals("io.webthings.app", appContext.packageName)
```

File: app/build.gradle [hunk 3]
```diff
@@ -0,0 +1,61 @@
+plugins {
+    id 'com.android.application'
+    id 'org.jetbrains.kotlin.android'
+}
+
+android {
+    compileSdk 31
+
+    defaultConfig {
+        applicationId "io.webthings.webthings"
```
Comment (tim-hellhake) [inline app/build.gradle:10]:
```suggestion
        applicationId "io.webthings.app"
```

**Review by tim-hellhake — CHANGES_REQUESTED** [review]

**Comment by arist0v** [comment]
> @arist0v You don't need to replicate the changes in the other PRs. As soon this is merged you can rebase the other branches.

yeah i knew, but i never done that and was kind of lazy googling :P

**Comment by arist0v** [comment]
should be good to go now

**Review by tim-hellhake — APPROVED** [review]
Just installed it on my phone and everything seems to be fine.

**Pull request merged**